src/components/Dashboard/FilingHistoryList/filings/LimitedRestorationExtension.vue

```diff
@@ -16,6 +16,20 @@
       </p>
 
       <ContactInfo class="mt-4" />
+
+      <p
+        v-if="fileNumber"
+        class="mt-4 mb-0"
+      >
+        Court Order Number: {{ fileNumber }}
+      </p>
+
+      <p
+        v-if="hasEffectOfOrder"
+        class="mt-0"
+      >
+        Pursuant to a Plan of Arrangement
+      </p>
     </template>
```

Comment:
It seems there is no place for the staff to choose a POA, shall we keep this part?
![image](https://github.com/user-attachments/assets/18a140a5-5028-46be-b6ee-8670ebb89a1d)

Reply:
Good catch. You are correct: there is no place to select "POA" in the limited restoration filing or extension filing. Therefore, yes, please delete lines 27-32.

Reply:
And also delete the getter below.